addons/web_editor/static/lib/odoo-editor/src/commands/commands.js

```diff
@@ -528,7 +528,7 @@ export const editorCommands = {
                 const block = closestBlock(node);
                 if (!['OL', 'UL'].includes(block.tagName)) {
                     const ublock = block.closest('ol, ul');
-                    ublock && getListMode(ublock) == mode ? li.add(block) : blocks.add(block);
+                    ublock && getListMode(ublock) == mode ? li.add(block) : block.parentNode.tagName === 'LI' ? blocks.add(block.parentNode) : blocks.add(block);
```

Comment:
Looking at your fix, I feel like the issue is that the current code does not handle blocks that are inside LI correctly. Could you please try this and tell me if it works ? Thanks !
```js
let block = closestBlock(node);
if (!['OL', 'UL'].includes(block.tagName)) {
    block = block.closest('li') || block;
    const ublock = block.closest('ol, ul');
```

Reply:
Sure.

Reply:
@dmo-odoo I tried the changes, and it's working fine. I have updated the code, please check.

Reply:
Awesome, thanks !